game-assets/cncnet.pack/rulesmd.ini

```diff
@@ -22594,7 +22594,7 @@ DieSound=BuildingMetalDamaged
 
 ;NK Flag Left; CCCP84 Nov 2025: it was in unused files. added to ini files and shp to the mix
 [CANKFGL]
-UIName=Name:NKOREAFLAG
+UIName=NOSTR:N.Korean Flag
```

Comment:
No, `NOSTR` strings are unacceptable for this config. If you want to add new strings, add them to the .csf file instead.